packages/vtm/netxs-os-system.hpp.patch

```diff
@@ -10,3 +10,10 @@ diff -uNr vtm-0.8.0/src/netxs/os/system.hpp vtm-0.8.0.mod/src/netxs/os/system.hp
                          auto size = socklen_t{ sizeof(cred) };
                      #else
                          auto size = unsigned{ sizeof(cred) };
+@@ -4052,4 +4052,4 @@
+             std::this_thread::sleep_for(200ms); // Pause to complete consuming/receiving buffered input (e.g. mouse tracking) that has just been canceled.
+         }
+     };
+-}
+\ No newline at end of file
++}
```

Comment:
Please remove this part of patch. This only adds a newline at the end of the file, and can be a cause of rejection with a future update.